src/operator/subgraph/build_subgraph.cc

```diff
@@ -101,12 +68,12 @@ void PrintNodeEntries(const std::vector<nnvm::NodeEntry*>& entries) {
  * \param g the MXNet computational graph
  * \param simple_nodes the nodes of undirected graph in top sorted order
  */
-void CreateSimpleGraph(const Graph& g,
-                       std::vector<SimpleNodePtr>* simple_nodes) {
+void CreateSimpleGraph(const nnvm::Graph& g,
+                       std::vector<BiDirectionalNodePtr>* simple_nodes) {
```

Comment:
BiDirectional -> BiDirected is more appropriate.

Reply:
Changed.